Integrations/integration-VirusTotal-Private_API.yml

```diff
@@ -61,6 +61,12 @@ script:
     # disable insecure warnings
     requests.packages.urllib3.disable_warnings()
 
+    if not demisto.params().get('proxy', False):
```

Comment:
Parameter name is `useProxy` and not `proxy`

Reply:
nice catch :)